compiler/AST/iterator.cpp

```diff
@@ -1159,7 +1235,11 @@ void lowerIterator(FnSymbol* fn) {
 
   BlockStmt* singleLoop = NULL;
   if (CallExpr* singleLoopYield = isSingleLoopIterator(fn, asts))
+  {
```

Comment:
Can you move this brace to same line to match the style in the rest of the function